DEVDOC.md

```diff
@@ -1,5 +1,43 @@
 # GAPID Developer Documentation
 
+## Setup Golang development
+
+This project contains Golang code, but it does not have the file hierarchy of
+regular Golang projects (this is due to the use of Bazel as a build system).
+The `cmd/gofuse` utility enables to re-create the file hierarchy expected by Go
+tools:
+
+```sh
+# We assume GAPID has been checked out under ~/gapid
+cd ${HOME}/gapid
+# Make sure to build to have all compile-time generated files
+bazel build //pkg
+# Prepare a gapid-gofuse directory **outside of the gapid checkout directory**
+mkdir ${HOME}/gapid-gofuse
+# Run gofuse with the previous directory as a target, and the bazel-specific OS identifier
+OS=k8 # Pick up the right option: k8 (linux) / darwin (macos) / x64_windows (windows)
+bazel run //cmd/gofuse -- -bazelout ${OS}-fastbuild --dir ${HOME}/gapid-gofuse
```

Comment:
I think the option is -dir with a single dash.
I would use here only
```bazel run //cmd//gofuse -- -dir <path-outside-gapid-source>/gapid-gofuse```
and let gofuse use the default for bazelout. If we describe the -bazelout option, we should also mention that the configuration can be fastbuild, dbg or opt, and that in doubt users can just check the folders under bazel-out/

Reply:
Done.

The gapid utility commands accept both one or two dash for options, but yes let's stick with what their help message displays.